### PR TITLE
feat: add traffic type of peer task download duration

### PR DESCRIPTION
**Pull request by litianxi**
feat: add traffic type of peer task download duration

### CODE REVIEW

File: scheduler/service/service_v1.go [hunk 1]
```diff
@@ -311,18 +311,19 @@ func (v *V1) ReportPeerResult(ctx context.Context, req *schedulerv1.PeerResult)
...
 		go v.createDownloadRecord(peer, parents, req)
 		v.handleTaskSuccess(ctx, peer.Task, req)
 		v.handlePeerSuccess(ctx, peer)
+		metrics.DownloadPeerDuration.WithLabelValues(priority.String(), peer.Task.Type.String(),
+			peer.Task.Tag, peer.Task.Application, peer.Host.Type.Name(), commonv2.TrafficType_BACK_TO_SOURCE.String()).Observe(float64(req.Cost))
 		return nil
 	}
 
+	metrics.DownloadPeerDuration.WithLabelValues(priority.String(), peer.Task.Type.String(),
+		peer.Task.Tag, peer.Task.Application, peer.Host.Type.Name(), commonv2.TrafficType_LOCAL_PEER.String()).Observe(float64(req.Cost))
```
Comment (gaius-qi) [inline scheduler/service/service_v1.go:325]: Change `commonv2.TrafficType_LOCAL_PEER` to `commonv2.TrafficType_REMOTE_PEER`.

**Review by gaius-qi — CHANGES_REQUESTED** [review]
Please remove the changes in the `scheduler/service/service_v2.go`. I will finish it.

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by gaius-qi — CHANGES_REQUESTED** [review]
Fix unit testing.
```text
2023-05-15T07:22:43.5857994Z --- FAIL: TestServiceV2_handleDownloadPeerFinishedRequest (0.00s)
2023-05-15T07:22:43.5858403Z     --- PASS: TestServiceV2_handleDownloadPeerFinishedRequest/peer_can_not_be_loaded (0.00s)
2023-05-15T07:22:43.5858899Z     --- PASS: TestServiceV2_handleDownloadPeerFinishedRequest/peer_state_is_PeerStateSucceeded (0.00s)
2023-05-15T07:22:43.5859335Z     --- FAIL: TestServiceV2_handleDownloadPeerFinishedRequest/peer_state_is_PeerStateRunning (0.00s)
2023-05-15T07:22:43.5859633Z panic: inconsistent label cardinality: expected 6 label values but got 5 in []string{"LEVEL0", "DFDAEMON", "d7y", "foo", "normal"} [recovered]
2023-05-15T07:22:43.5859958Z 	panic: inconsistent label cardinality: expected 6 label values but got 5 in []string{"LEVEL0", "DFDAEMON", "d7y", "foo", "normal"} [recovered]
2023-05-15T07:22:43.5860256Z 	panic: inconsistent label cardinality: expected 6 label values but got 5 in []string{"LEVEL0", "DFDAEMON", "d7y", "foo", "normal"}
```

**Comment by litianxi** [comment]
> Fix unit testing.
> 
> ```
> 2023-05-15T07:22:43.5857994Z --- FAIL: TestServiceV2_handleDownloadPeerFinishedRequest (0.00s)
> 2023-05-15T07:22:43.5858403Z     --- PASS: TestServiceV2_handleDownloadPeerFinishedRequest/peer_can_not_be_loaded (0.00s)
> 2023-05-15T07:22:43.5858899Z     --- PASS: TestServiceV2_handleDownloadPeerFinishedRequest/peer_state_is_PeerStateSucceeded (0.00s)
> 2023-05-15T07:22:43.5859335Z     --- FAIL: TestServiceV2_handleDownloadPeerFinishedRequest/peer_state_is_PeerStateRunning (0.00s)
> 2023-05-15T07:22:43.5859633Z panic: inconsistent label cardinality: expected 6 label values but got 5 in []string{"LEVEL0", "DFDAEMON", "d7y", "foo", "normal"} [recovered]
> 2023-05-15T07:22:43.5859958Z 	panic: inconsistent label cardinality: expected 6 label values but got 5 in []string{"LEVEL0", "DFDAEMON", "d7y", "foo", "normal"} [recovered]
> 2023-05-15T07:22:43.5860256Z 	panic: inconsistent label cardinality: expected 6 label values but got 5 in []string{"LEVEL0", "DFDAEMON", "d7y", "foo", "normal"}
> ```

It failed the service v2's unit test, it seems that if the service v2's logic is not modified, it will be affected

**Pull request merged**